setup.py

```diff
@@ -41,7 +41,7 @@
         "Framework :: Wagtail",
         "Framework :: Wagtail :: 2",
     ],
-    install_requires=["Django>=2.2,<4.0", "Wagtail>=2.11,<2.16", "polib>=1.1,<2.0"],
+    install_requires=["Django>=2.2,<4.0", "Wagtail>=2.11,<2.17", "polib>=1.1,<2.0"],
```

Comment:
This is probably the more controversial change. In order to run the tests agains Wagtail main, we want the upper bound to be vNext+1

Reply:
I think we should avoid doing this here since there's a good chance we could accidentally ship a package that falsely advertises 2.16 support and cause issues later.

Reply:
any suggestion on how to work around https://github.com/wagtail/wagtail-localize/runs/4079597377?check_suite_focus=true

```
ERROR: Cannot install wagtail 2.16a0 (from git+https://github.com/wagtail/wagtail.git) and wagtail-localize[testing]==1.0rc4 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested wagtail 2.16a0 (from git+https://github.com/wagtail/wagtail.git)
    wagtail-localize[testing] 1.0rc4 depends on Wagtail<2.16 and >=2.11
```